asv_bench/asv.conf.json

```diff
@@ -36,7 +36,7 @@
     "install_timeout": 600,
 
     // the base URL to show a commit for the project.
-    // "show_commit_url": "http://github.com/owner/project/commit/",
```

Comment:
I think this is already on master, can we roll this back?

Reply:
I've rebased onto master some point.  Reverted it now.